qiskit_nature/operators/second_quantization/spin_op.py

```diff
@@ -148,53 +148,46 @@ class SpinOp(ParticleOp):
     """
 
     def __init__(
-            self,
-            data: Union[
-                str,
-                List[Tuple[str, complex]],
-                Tuple[np.ndarray, np.ndarray],
-            ],
-            spin: float = 1 / 2,
-            label_mode: str = "sparse",
+        self,
+        data: Union[
+            str,
+            List[Tuple[str, complex]],
+            Tuple[np.ndarray, np.ndarray],
+        ],
+        spin: float = 1 / 2,
+        dtype: Union[type, str] = np.complex128,
     ):
         r"""Initialize ``SpinOp``.
```

Comment:
I see. Hou about this?
<img width="350" alt="image" src="https://user-images.githubusercontent.com/6814928/109900386-a2bde200-7cda-11eb-833f-ca6ddf80b22b.png">

Reply:
I meant we end up with text like you show there which would come just after the docstring from the class and these two words like 'Initialise SpinOp' or 'SpinOp Constructor' etc really add no value and often clutter the text. It should be self evident since this is the documentation for the init method, i,e the constructor method of a class, such that text like this is not needed.

Reply:
Ah, I understood it. Thanks.